types/react-autosuggest/index.d.ts

```diff
@@ -205,6 +203,12 @@ declare namespace Autosuggest {
          * Use your imagination to define how suggestions are rendered.
          */
         renderSuggestion: RenderSuggestion<TSuggestion>;
+        /**
+         * When a suggestion is selected, Autosuggest will consult this function on whether to close the suggestions list.
+         * Use it, for example, if you want to make multiple selections at the same time. By default it will close every time
+         * a suggestion is selected.
+         */
+        shouldKeepSuggestionsOnSelect?: ShouldKeepSuggestionsOnSelect<TSuggestion>;
```

Comment:
This is the main change that we needed, to be able to use `shouldKeepSuggestionsOnSelect` within our project.